JavaFiddle-ejb/src/main/java/Test.java

```diff
@@ -0,0 +1,67 @@
+
+
+import ru.javafiddle.core.ejb.ProjectBean;
+import ru.javafiddle.jpa.entity.*;
+import ru.javafiddle.core.ejb.UserBean;
+import ru.javafiddle.core.ejb.GroupBean;
+
+import javax.faces.event.SystemEvent;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+import java.io.UnsupportedEncodingException;
+import java.security.NoSuchAlgorithmException;
+
+
+/**
+ * Created by artyom on 06.02.16.
+ */
+public class Test {
```

Comment:
Yes, it should have been deleted long time ago.

воскресенье, 17 апреля 2016 г. пользователь alexander1leonov написал:

> In JavaFiddle-ejb/src/main/java/Test.java
> <https://github.com/roma-turkin/javafiddle/pull/34#discussion_r59989556>:
>
> > +import ru.javafiddle.jpa.entity.*;
> > +import ru.javafiddle.core.ejb.UserBean;
> > +import ru.javafiddle.core.ejb.GroupBean;
> > +
> > +import javax.faces.event.SystemEvent;
> > +import javax.persistence.EntityManager;
> > +import javax.persistence.EntityManagerFactory;
> > +import javax.persistence.Persistence;
> > +import java.io.UnsupportedEncodingException;
> > +import java.security.NoSuchAlgorithmException;
> > +
> > +
> > +/**
> > + * Created by artyom on 06.02.16.
> > + */
> > +public class Test {
>
> Do we need this class in our repo at all? I strongly doubt.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/roma-turkin/javafiddle/pull/34/files/d9efe0c42b1ecf174a709400ed43661476b30e52#r59989556>
>